src/java.base/share/classes/java/text/Format.java

```diff
@@ -61,9 +61,9 @@
  * <h2>Subclassing</h2>
  *
  * <p>
- * The Java Platform provides three specialized subclasses of {@code Format}--
- * {@code DateFormat}, {@code MessageFormat}, and
- * {@code NumberFormat}--for formatting dates, messages, and numbers,
+ * The Java Platform provides four specialized subclasses of {@code Format}--
```

Comment:
I would drop the "four" so its easier to maintain later; the sentence reads fine without it.